src/content/en/2020/resource-hints.md

```diff
@@ -8,16 +8,336 @@ authors: [Zizzamia]
 reviewers: [notwillk, giopunt, jessnicolet, pmeenan, mgechev]
 analysts: [khempenius]
```

Comment:
Can you verify that all of these reviewers/analysts have meaningfully contributed to the chapter and are roughly in sorted order so that the biggest contributors are listed first?